cmd/hyperkube/kubectl.go

```diff
@@ -17,14 +17,15 @@ limitations under the License.
 package main
 
 import (
-	"os"
-
+	"github.com/docker/docker/pkg/term"
 	"k8s.io/kubernetes/pkg/kubectl/cmd"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 )
 
 func NewKubectlServer() *Server {
-	cmd := cmd.NewKubectlCommand(cmdutil.NewFactory(nil), os.Stdin, os.Stdout, os.Stderr)
+	// need to use term.StdStreams to get the right IO refs on Windows
```

Comment:
I did it here for hyperkube and also below for kubectl. I did it here because I figured the top-down approach made the most sense, instead of specifically going into the exec/run/attach commands and ignoring what was passed in for stdout and overriding it with the value from term.StdStreams.